hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/AvroConversionHelper.scala

```diff
@@ -314,7 +313,7 @@ object AvroConversionHelper {
           } else {
             val sourceArray = item.asInstanceOf[Seq[Any]]
             val sourceArraySize = sourceArray.size
-            val targetList = new util.ArrayList[Any](sourceArraySize)
+            val targetList = new java.util.ArrayList[Any](sourceArraySize)
```

Comment:
back these changes out?

Reply:
Reverted these changes, but for some reason my intellij complains and cannot find util.Arraylist etc